go.mod

```
@@ -1,24 +1,24 @@
 module github.com/minio/hperf
 
-go 1.22
+go 1.22.4
```

```suggestion
go 1.22
```

Whatever script you have that does this automatically, kill it.

---

it's just go mod init and tidy